mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -181,6 +182,7 @@ properties:
           - :RELEASE
           - :SNAPSHOT
         default_value: :VERSION_POLICY_UNSPECIFIED
```

Comment:
Ah- X_UNSPECIFIED values don't ever get returned by the REST API, so this would have needed special handling in the first place. Generally we omit them as enum values at all, because sending nothing does the same thing & null (as in lack of presence) is what the API will return. Given that we added the value + made it the default, `default_if_empty` is the most reasonable way to address this.